tests/test_filtering.py

```diff
@@ -1160,6 +1160,7 @@ def filter_username(self, queryset, name, value):
                 return queryset.filter(**{name: value})
 
         self.assertEqual(list(F().qs), list(User.objects.all()))
+        self.assertEqual(list(F(queryset=User.objects.all())), list(User.objects.all()))
```

Comment:
btw, the filterset is no longer directly iterable. This was removed in #440

`F(queryset=User.objects.all())` => `F(queryset=User.objects.all()).qs`